recipes/lfortran/build.sh

```diff
@@ -7,8 +7,7 @@ printenv
 if [[ ${MACOSX_DEPLOYMENT_TARGET} != "" ]]; then
     # macOS
     python -m pip install . -vv \
-      --install-option="build" \
-      --install-option="--" \
+      --install-option="build_ext" \
       --install-option="-DCMAKE_OSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET}"
```

Comment:
Are you sure the previous patch didn't work? The build got canceled. I think you have to put in the `--`.